LSPS0/README.md

```diff
@@ -391,6 +379,66 @@ SHOULD indicate this as an "unrecognized" error to the user.
 > then use to help guide the evolution of the specification, or
 > to comply with later versions of the LSPS spec.
 
+##### `-32602` Invalid Parameters Error
+
+An LSP that sends back an invalid parameter error with
+`code = -32602` MUST include a `data` object in the `error`
+response.
+
+This `data` object MUST contain at least the field `unrecognized`,
+a JSON array of strings, representing an unordered set of
+parameter names.
+These are the parameter names that are not recognized by the LSP as
+valid for the RPC method.
+
+For example, suppose the client sent this request:
+
+```JSON
+{
+  "jsonrpc": "2.0",
+  "method": "example.method_name",
+  "params": {
+    "v1_parameter": "value1",
+    "v2_parameter": "value2"
+  },
+  "id": "42"
+}
+```
```

Comment:
Can we rephrase to speak in terms of `features` and not in terms of `versions`?

It is somewhat confusing to use the wording versions here and explain in the last section that we don't want to support a versioning scheme.



```JSON
{
  "jsonrpc": "2.0",
  "method": "example.method_name",
  "params": {
    "v1_feature": "value1",
    "v2_feature": "value2"
  },
  "id": "42"
}
```